bin/sandbox

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env ruby
 
-require_relative '../lib/mruby_engine'
+require "pathname"
+ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
+  Pathname.new(__FILE__).realpath)
```

Comment:
Just for my understanding, this is the equivalent of 
```
ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", File.dirname(Pathname.new(__FILE__).realpath))
```
i.e. `../../` is just going two level up from the _file_, rather than the file's location. Honestly if this works, it's all fine with me... Just surprised we're navigating from a file (vs. a directory). I guess these operations are not FS dependent?